Dockerfile

```diff
@@ -1,8 +1,29 @@
 # syntax=docker/dockerfile:1
 
+## Buildstage ##
+FROM ghcr.io/linuxserver/baseimage-alpine:3.19 AS buildstage
+
+RUN curl -L https://archive.org/download/kindlegen_linux_2_6_i386_v2_9/kindlegen_linux_2.6_i386_v2_9.tar.gz > kindlegen.tar.gz
+RUN tar -zxvf kindlegen.tar.gz kindlegen
+RUN chmod +rwx 'kindlegen'
+RUN rm kindlegen.tar.gz
+
+RUN latest_tag=$(curl -s https://api.github.com/repos/ciromattia/kcc/releases/latest | jq -rc ".tag_name") && \
```

Comment:
Please use the var name `MOD_VERSION` and check whether it's passed in and ARG like here: https://github.com/linuxserver/docker-mods/blob/transmission-transmissionic/Dockerfile#L5-L13